doc/Overview.rst

```diff
@@ -1,13 +1,19 @@
 Overview
 ================
 
+Human And Robot Modular OpeN Interactions (HARMONI) is a comprehensive tool containing all the components and capabilities you need to quickly get your social interaction up and running on a robot. 
+
+HARMONI packages the additional capabilities needed for social human-robot interaction neatly on top of ROS by wrapping and integrating several state of the art libraries in the domains of natural language understanding, dialog, object and face detection, and decision trees. HARMONI is the glue that integrates these disparate packages on top of ROS in a standardized way so you don't have to. Just `install our docker images <quickstart/Docker-Quickstart.html>`_ and `configure the capabilities you need <quickstart/Multi-Container.html>`_ for your interaction.
```

Comment:
Is the second link supposed to link to configuration? Configuration.html instead of quickstart/Multi-Container? Or is it actually the container configuration, in which case it should be tutorials/Multi-Container.html

Reply:
Good catch. Yes, configuration link. Updated.